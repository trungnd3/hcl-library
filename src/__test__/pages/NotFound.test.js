import React from 'react';
import NotFound from '../../pages/NotFound';
import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';

describe('NotFound component using RTL', () => {
  test('render snapshot', () => {
    const tree = renderer.create(<NotFound />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  test('render NotFound', () => {
    render(<NotFound />);
    const notFoundText = screen.getByText('Page not found!');
    expect(notFoundText).toBeInTheDocument();
  });
});
