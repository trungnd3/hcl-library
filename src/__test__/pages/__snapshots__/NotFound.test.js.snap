// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound component using RTL render snapshot 1`] = `
<div
  className="mt-5 text-center"
>
  <h1>
    Page not found!
  </h1>
</div>
`;
